assets/js/googlesitekit/datastore/user/key-metrics.js

```diff
@@ -257,6 +257,26 @@ const baseSelectors = {
 		return keyMetricsSettings.widgetSlugs;
 	} ),
 
+	/**
+	 * Gets whether an individual key metric identified by its slug is
+	 * active or not.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {boolean|undefined} True if the key metric widget tile is active, false if it is not, or undefined if the key metrics settings are not loaded.
+	 */
+	isKeyMetricActive: createRegistrySelector(
```

Comment:
Note, every module asset that uses this should be modified to depend on the core/user store's asset now. I can see at least GA4 currently does not
https://github.com/google/site-kit-wp/blob/37352fca946b60a142a4aea0cc19d0e056fc9285/includes/Modules/Analytics_4.php#L933-L945